tests/trestlebot/cli/test_create_cmd.py

```diff
@@ -0,0 +1,149 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Red Hat, Inc.
+
+""" Unit test for create commands ssp and cd"""
+
+from click.testing import CliRunner
+
+from trestlebot.cli.commands.create import create_cmd
+
+
+def test_invalid_create_cmd() -> None:
+    """Tests that create command fails if given invalid oscal model subcommand."""
+    runner = CliRunner()
+    result = runner.invoke(create_cmd, ["invalid"])
+
+    assert "Error: No such command 'invalid'" in result.output
+    assert result.exit_code == 2
+
+
+def test_create_ssp_cmd(tmp_init_dir: str) -> None:
+    """Tests successful create ssp command."""
+    compdef_list_tester = ["ac-12", "ac-13"]
+    ssp_index_tester = "tmp_init_dir/tester-ssp-index.json"
+
+    runner = CliRunner()
+    result = runner.invoke(
+        create_cmd,
+        [
+            "ssp",
+            "--profile-name",
+            "oscal-profile-name",
+            "--markdown-dir",
+            "markdown",
+            "--compdefs",
+            compdef_list_tester,
+            "--ssp-name",
+            "test-name",
+            "--repo-path",
+            tmp_init_dir,
+            "--ssp-index-file",
+            ssp_index_tester,
+        ],
+    )
+    assert result.exit_code == 0
+
+
+def test_create_compdef_cmd(tmp_init_dir: str) -> None:
+    """Tests successful create compdef command."""
+
+    runner = CliRunner()
+    result = runner.invoke(
+        create_cmd,
+        [
+            "compdef",
+            "--profile-name",
+            "oscal-profile-name",
+            "--compdef-name",
+            "test-name",
+            "--component-title",
+            "title-test",
+            "--component-description",
+            "description-test",
+            "--component-definition-type",
+            "type-test",
+            "repo-path",
+            tmp_init_dir,
+        ],
+    )
+    assert result.exit_code == 2
```

Comment:
The expected exit_code is 2? Is there any context I missed?

Reply:
I think the arguments, `'--committer-email'`, `'--committer-name'` and `'--branch'`could be added and the `assert result.exit_code == 2` could be `result.exit_code == 0 `.

Reply:
@huiwangredhat yes, you are correct, this should have a successful exit code since test expects it work correctly.